src/main/java/com/training/springpractice/AppConfig.java

```diff
@@ -0,0 +1,15 @@
+package com.training.springpractice;
+
+import org.modelmapper.ModelMapper;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+@ComponentScan({"com.training.springpractice"})
```

Comment:
Maybe it's not necessary to use this annotation, since your SpringPracticeApplication has **@SpringBootApplication** annotation, please take a look at that main annotation in order to understand if the ComponentScan is necessary or not and tell me your findings.